scripts/ipfs/install

```diff
@@ -5,7 +5,10 @@ set -e
 GO_IPFS_VERSION=v0.4.6
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-
+# Check for PI user if missing add it
+if [ -z $(cat /etc/passwd | grep pi:x) ];then
+    useradd pi
```

Comment:
I looked at that but some of the configs that get copied over are hardcoded to /home/pi/.ipfs  
This is why you need the pi user.  Perhaps a revisit of this is in order but i figured this was a quick patch.